bin/cyhy-kevsync

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+
+"""Parse KEV (Known Exploited Vulnerabilities) JSON file and save to database.
+
+Usage:
+  cyhy-kevsync [--section SECTION]
+  cyhy-kevsync [--section SECTION] --local-file <json-file>
+  cyhy-kevsync (-h | --help)
+  cyhy-kevsync --version
+
+Options:
+  -h --help                      Show this screen.
+  -f --local-file                Import KEV data from a local file.
+  --version                      Show version.
+  -s SECTION --section=SECTION   Configuration section to use.
+
+"""
+
+# standard python libraries
+import json
+from StringIO import StringIO
+import urllib
+
+# third-party libraries (install with pip)
+from docopt import docopt
+
+# intra-project modules
+from cyhy.db import database
+from cyhy.util import util
+
+KEV_URL = "https://www.cisa.gov/sites/default/files/feeds/known_exploited_vulnerabilities.json"
+
+
+def remove_outdated(db, imported_cves):
+    # Find KEV CVEs in DB that were NOT imported just now
+    outdated_kevs = db.KEVDoc.find({"_id": {"$nin": imported_cves}})
+    outdated_cves = [i["_id"] for i in outdated_kevs]
+
+    if outdated_cves:
+        # Remove outdated CVEs from KEV collection
+        db.KEVDoc.collection.remove({"_id": {"$in": outdated_cves}})
+
+        print("The following CVEs were removed from the KEV collection:")
+        print("\n".join(outdated_cves))
+
+
+def parse_json(db, json_stream):
+    data = json.load(json_stream)
+
+    # Gather all the cveIDs from the JSON file
+    all_cve_ids = set()
+    for i in data.get("vulnerabilities"):
```

Comment:
If `vulnerabilities` is missing I think a `KeyError` is more useful than a `TypeError` when checking the log. Alternatively a check like that done for `cveID` values below could be done to provide a more specific error.

```suggestion
    for i in data["vulnerabilities"]:
```

Reply:
I agree with you that a `KeyError` will be more useful. 👍